app/Http/Controllers/UserController.php

```diff
@@ -23,16 +23,18 @@ public function store(Request $request)
         $request->validate([
             'first_name' => 'required',
             'last_name' => 'required',
-            'email' => 'required',
+            'email' => 'required|unique:users',
```

Comment:
```suggestion
            'email' => 'required|email|unique:users',
```
Will check if this is a valid email:  `test@test` will pass because *technically*, it's a valid email address. <https://en.wikipedia.org/wiki/Email_address#Examples>

But if you want, you can add `email:dns` to tell PHP to check if the domain is really valid. There is other modifier on the email rule in the doc : <https://laravel.com/docs/8.x/validation#rule-email>